internal/controllers/requirements/docker/docker.go

```diff
@@ -104,15 +92,7 @@ func validateIfDockerIsRunningInMinVersion(response string) error {
 }
 
 func extractDockerVersionFromString(response string) (int, int, error) {
-	responseSpited := strings.Split(strings.ToLower(response), "docker version ")
-	if len(responseSpited) < 1 || len(responseSpited) > 1 && len(responseSpited[1]) < 8 {
-		return 0, 0, ErrDockerNotInstalled
-	}
-	return getVersionAndSubVersion(responseSpited[1])
-}
-
-func checkIfContainsDockerVersion(response string) bool {
-	return strings.Contains(strings.ToLower(response), "docker version ")
+	return getVersionAndSubVersion(response)
```

Comment:
We can remove `extractDockerVersionFromString` function and use directly `getVersionAndSubVersion`